### ISSUE TITLE
🐛 Bug Report: Project Logo preview doesn't work in the Console

### ISSUE THREAD

**Issue by karniv00l** [appwrite/appwrite#3054]
### 👟 Reproduction steps

- Create a new project
- Go to Project Settings and upload Project Logo
- Refresh the page
- Failed HTTP GET made in the browser looks like this: 

```
/v1/storage/buckets/undefined/files/undefined/preview?width=200&height=200&project=console&mode=admin
```

### 👍 Expected behavior

The uploaded Project Logo should be visible

### 👎 Actual Behavior

The uploaded Project Logo is not visible on the Settings page

### 🎲 Appwrite version

Version 0.13.x

### 💻 Operating system

Linux

### 🧱 Your Environment

Happens both on local macOS env as on Linux VPS

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by Meldiron** [comment]
Great catch! I have addressed the issue in a PR 🥳

**Comment by eldadfux** [comment]
This feature is now released and available as part of Appwrite 0.14. More on the new release here: https://github.com/appwrite/appwrite/discussions/3240

**Comment by eldadfux** [comment]
This feature is now released and available as part of Appwrite 0.14. More on the new release here: https://github.com/appwrite/appwrite/discussions/3240
